src/docker_compose.rs

```diff
@@ -178,8 +178,8 @@ fn generate_docker_compose_content(dockerfile: PathBuf, dev_conf: &Option<Dev>)
                                 "-q".into(),
                                 "http://localhost:4000".into(),
                             ])),
-                            interval: Some("10s".into()),
-                            start_period: Some("5s".into()),
```

Comment:
Why did you raise the start_period for the devtools ? I don't think this is a good idea because depending on the computer speed, the devtools can start in less than 10 seconds. You should just add more retries.

Reply:
To let devtool time to start because depending on the speed of the computer devtool can take up to 9s to start.

Reply:
There is two problems to making more retries: it uses more resources and it displays to much logs.
We should create a specific end point to check that the server available without logging and without calculating anything.